datalad/core/distributed/push.py

```diff
@@ -767,11 +771,14 @@ def _push_data(ds, target, content, force, jobs, res_kwargs,
     if jobs:
         cmd.extend(['--jobs', str(jobs)])
 
-    if not to_transfer and force not in ('all', 'datatransfer'):
-        lgr.debug("Invoking copy --auto")
-        cmd.append('--auto')
+    if force not in ('pushall', 'datatransfer') and ds_repo.config.obtain(
```

Comment:
nit-pick: Why not use `getbool` here?

Reply:
I dislike the compatibility interfaces that require splitting the config key into section and option name.